Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Distributions"
 uuid = "31c24e10-a181-5473-b8eb-7969acd0382f"
 authors = ["JuliaStats"]
-version = "0.25.41"
+version = "0.25.42"
```

Comment:
Should we bump the StatsFuns compat to ensure that https://github.com/JuliaStats/StatsFuns.jl/pull/132 is fixed and users don't run into this issue?

Reply:
Yes I think so, once it's released.